x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -284,6 +298,20 @@ export const CommandInput = memo<CommandInputProps>(({ prompt = '', focusRef, ..
             </EuiFlexGroup>
             <InputPlaceholder />
           </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonIcon
+              data-test-subj="inputTextSubmitButton"
+              aria-label="submit-command"
+              iconType="playFilled"
+              display="empty"
+              color="primary"
+              isDisabled={useMemo(
```

Comment:
suggestion: pull this `useMemo()` up so that it is not inside of the `return` JSX